src/main/java/core/basesyntax/file/FileReaderImpl.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax.file;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+public class FileReaderImpl implements FileReader {
+    @Override
+    public List<String> read(String filePath) {
+        List<String> lines = new ArrayList<>();
+        try (BufferedReader reader = new BufferedReader(new java.io.FileReader(filePath))) {
+            StringBuilder builder = new StringBuilder();
+            String line = reader.readLine();
+            while (line != null) {
+                lines.add(line);
+                line = reader.readLine();
+            }
+        } catch (IOException e) {
+            throw new RuntimeException("Can't read file" + filePath, e);
```

Comment:
The error message should provide more context and be more informative. Include a space before the file path for readability.